README.md

```diff
@@ -20,3 +24,10 @@ For example, the dataset can now be accessed as "https://raw.githubusercontent.c
 ```
 
 Hence, it will no longer be necessary to upload datasets into Google Drive and mount the drive in Colab notebooks.
+
+# Acknowledgments / Credits / References
+* TLDRNews - obtained from [https://huggingface.co/datasets/JulesBelveze/tldr_news](https://huggingface.co/datasets/JulesBelveze/tldr_news), thanks to a gracious permission by the author, [Jules Belveze](https://www.linkedin.com/in/jules-belveze/).
+* LegalClause - obtained from [Legal Clause Type Hint Removed From Clause Text](https://www.kaggle.com/datasets/alexandersherstinsky/legal-clause-type-hint-removed-from-clause-text), which is derived from [Legal-Clause-Dataset](https://www.kaggle.com/datasets/bahushruth/legalclausedataset), thanks to a gracious permission by the author, Bunty ([@Bahushruth](https://twitter.com/Bahushruth)).
+* Magicoder_OSS_Instruct_75K - obtained from [https://huggingface.co/datasets/ise-uiuc/Magicoder-OSS-Instruct-75K](https://huggingface.co/datasets/ise-uiuc/Magicoder-OSS-Instruct-75K), thanks to a gracious permission by the author, [Yuxiang Wei ](https://www.linkedin.com/in/yuxiang-wei-a94a63205/).
+* CoNLL-2003 - obtained from [https://huggingface.co/datasets/conll2003](https://huggingface.co/datasets/conll2003); reached out to the author, [Erik Tjong Kim Sang](https://www.linkedin.com/in/erik-tjong-kim-sang-5a36205/), for permission (on 2/29/2024); awaiting response. 
```

Comment:
Should we just add CoNLL-2003 once we have permission?

Reply:
@arnavgarg1 I feel that we can keep it for now (since the dataset is public on HuggingFace).  In addition, I am monitoring for a response from the author; hoping that they will agree, as did the others.  However, if they decline, then I will delete it.  Thank you.  /cc @w4nderlust